MOBILE_API.xml

```diff
@@ -1299,6 +1305,18 @@
                 The estimate range in KM the vehicle can travel based on fuel level and consumption.
             </description>
         </param>
+        <param name="level" type="Float" minvalue="-6" maxvalue="1000000" mandatory="false" since="6.0">
```

Comment:
@ydementieiev How fuel level can be negative?

Reply:
@AByzhynar  [this value existed before](https://github.com/smartdevicelink/rpc_spec/pull/226/files#diff-745a89443809bf31b6e5533025d37475L8055), I just renamed it and moved it to another place